From 7c81c2f6794c85b59a630febfb627e32fbd18f2f Mon Sep 17 00:00:00 2001
From: Gonzalo Odiard <godiard@gmail.com>
Date: Wed, 19 Feb 2014 13:58:05 -0300
Subject: [PATCH] The updater need reset the internal state when finish with
 error

This is needed to allow the next update request to work.

Signed-off-by: Gonzalo Odiard <godiard@sugarlabs.org>
---
 src/jarabe/model/update/updater.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/jarabe/model/update/updater.py b/src/jarabe/model/update/updater.py
index eac9101..3a6b990 100644
--- a/src/jarabe/model/update/updater.py
+++ b/src/jarabe/model/update/updater.py
@@ -129,6 +129,7 @@ class Updater(GObject.GObject):
     def _backend_error_cb(self, error):
         _logger.debug("_backend_error_cb %s", error)
         self._finished(True)
+        self._state = STATE_CHECKED
         self.emit('error', error)
 
     def update(self, bundle_ids):
-- 
1.8.1.4

